mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala

```diff
@@ -96,23 +96,8 @@ private[ml] trait DecisionTreeParams extends PredictorParams {
     " algorithm will cache node IDs for each instance. Caching can speed up training of deeper" +
     " trees.")
 
-  /**
```

Comment:
This scaladoc may still be useful, can we keep it around (maybe as the scaladoc for `setCheckpointInterval`)?